app/mailers/activity_mailer.rb

```diff
@@ -0,0 +1,19 @@
+class ActivityMailer < ApplicationMailer
+  layout 'activities/email'
+
+  # Send email to a recipient using input template URL and object
+  #
+  # @param [User] recipient the user who will receive this email
+  # @param sender the object which triggers off an activity and requires to send out an email
+  # @param object the tracked object in an activity and it will be used in the given template
+  # @param [String] template_url a string which is the path of email template
```

Comment:
`the path of the template` would be enough. Please try to make other documentation more concise as well.